### DISCUSSION TITLE
购买了专业版，一直未收到验证码

### DISCUSSION THREAD

**Discussion by juzhier** [getlantern/lantern-forum-cn#773] [Q&A] [answered]
8月8日从蓝灯主页在线支付购买了一年的专业版，但是授权设备使用专业版的验证码一直未收到，请管理员尽快回复帮忙解决！

**Comment by JinGuangyuan** [comment, marked as answer]
我也是同样的问题，8月3日购买，1年专业版，邮箱授权设备验证码无法收到，反馈多次还未回复

**Comment by MaxMKMax** [comment]
@juzhier 您好. 请您提供我您的账号邮箱和支付单据. 我帮您提交问题上去. 过几天去查看这个帖子或邮箱. 谢谢您!

**Reply by juzhier** [reply]
你好，我的账号邮箱是leecx23184260@gmail.com ，但这个邮箱用VPN也不太容易打开，可否将内容发送至我的另一个邮箱23184260@qq.com。支付单据如图
![微信图片_20240813083611](https://github.com/user-attachments/assets/2b90cb9d-e6f7-4912-84d6-f737ab803d33)

**Comment by MaxMKMax** [comment]
@JinGuangyuan 您好. 请您提供我您的账号邮箱和支付单据. 我帮您提交问题上去. 过几天去查看这个帖子或邮箱. 谢谢您!

**Reply by JinGuangyuan** [reply]
请问什么时候可以解决，账号邮箱和单据早已回复至邮箱

**Comment by juzhier** [comment]
***@***.*** ，但这个邮箱用VPN也不太容易打开，可否将内容发送至我的另一个邮箱
***@***.***。支付单据如图
[image: 微信图片_20240813083611.jpg]

Mike K ***@***.***> 于2024年8月11日周日 21:46写道：
